app/src/main/java/com/it/hungvt/movieapp/data/source/local/DatabaseHelper.java

```diff
@@ -0,0 +1,45 @@
+package com.it.hungvt.movieapp.data.source.local;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+
+/* Create database name */
+public class DatabaseHelper extends SQLiteOpenHelper {
+
+    private static final String DATA_BASE_NAME = "MyFavoriteMovie.db";
+    private static int DATA_BASE_VERSION = 1;
```

Comment:
WARNING: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName